tests_phi/pages/dashboard_page.py

```diff
@@ -0,0 +1,79 @@
+from selenpy.element.base_element import BaseElement
+from selenpy.element.text_box import TextBox
+from selenpy.element.combo_box import ComboBox
+from selenpy.element.check_box import CheckBox
+from tests_phi.utils.common_methods import CommonMethods
+from selenpy.support import browser
+
+class DashboardPage():
+    
+    common_methods = CommonMethods()
+    
+    _lbl_login_title = BaseElement("//div[@class='ltext' and text()='Login']")
+    _lbl_welcome_user = BaseElement("//a[@href='#Welcome']")
+    _lbl_logout = BaseElement("//a[@href='logout.do']")
+    _ico_global_setting = BaseElement("//li[@class='mn-setting']/a")
+    _txt_page_name = TextBox("id=name")
+    _cbb_parent_name = ComboBox("id=parent")
+    _cbb_num_of_column = ComboBox("id=columnnumber")
+    _cbb_display_after = ComboBox("id=afterpage")
+    _cb_public = CheckBox("id=ispublic")
+    _btn_dynamic_on_add_edit_new_page = BaseElement("//input[@class='button page_button' and @id='%s']")
+    _lbl_test_module_execution = BaseElement("//div[@title='Test Module Execution']")
+
+    def __init__(self):
+        pass
+
+    def logout(self):
+        self._lbl_welcome_user.wait_for_visible()
+        self._lbl_welcome_user.click()
+        self._lbl_logout.wait_for_visible()
+        self._lbl_logout.click()
+        self._lbl_welcome_user.wait_for_invisible()
+
+    def add_page(self, page_name, parent_page = "Overview"):
+        self.select_global_selecting_item("Add Page")
+        self.enter_page_info(page_name, parent_page)
+        self.click_button_on_add_edit_modal("OK")
+        self._txt_page_name.wait_for_invisible()
+        
+    def edit_page(self,page_name, parent_page = "Overview"):
+        self.select_global_selecting_item("Edit")
+        self.enter_page_info(page_name, parent_page)
+        self.click_button_on_add_edit_modal("OK")
+        self._txt_page_name.wait_for_invisible()
+        
+    def click_button_on_add_edit_modal(self, button):
+        self._btn_dynamic_on_add_edit_new_page.format(button)
+        self._btn_dynamic_on_add_edit_new_page.wait_for_visible()
+        self._btn_dynamic_on_add_edit_new_page.click()
+        
+    def enter_page_info(self,page_name, parent_page = "Overview"):
+        self._txt_page_name.wait_for_visible()
+        self._txt_page_name.find_element().clear()
+        self._txt_page_name.send_keys(page_name)
+        self._cbb_parent_name.wait_for_visible()
+        self._cbb_parent_name.select_by_text_contains(parent_page)
+           
+    def select_global_selecting_item(self, item):
+        self._ico_global_setting.wait_for_visible()
+        self._ico_global_setting.click()
+        self.common_methods.select_menu(item)
+        
+    def open_page(self,page_name):
```

Comment:
`def open_page(self,page_name)` should  `def open_page(self,*page_name) ` and we can use `open_page(page_1,page_2,page_3)`

Reply:
updated